CalorieTracker/CalorieTracker/Model/CoreDataStack.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CoreDataStack.swift
+//  CalorieTracker
+//
+//  Created by Kat Milton on 8/23/19.
+//  Copyright © 2019 Kat Milton. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+class CoreDataStack {
+    static let shared = CoreDataStack()
+    func save(context: NSManagedObjectContext = CoreDataStack.shared.mainContext) throws {
```

Comment:
This is great! Because you created the save method here, you won't have to remember to call `performAndWait` when you want to save the context to CoreData.